rector.php

```diff
@@ -17,6 +17,6 @@
 
     // define sets of rules
     $rectorConfig->sets([
-        LevelSetList::UP_TO_PHP_80,
+        LevelSetList::UP_TO_PHP_81,
```

Comment:
Do it in PR with PHP version bumping. Current minimum version is 8.0

Reply:
Reverted